language-design.md

```diff
@@ -17,7 +12,41 @@ For each quote, describe what it was about the quote that led you pick it.
+
+In today's era of quickly shifting standards and models, where every year some hot new language or approach to design comes out, we need to be more flexible with how we use languages, and how and why we design new ones.  When we think of language design as a tool for making more tools, we greatly enhance our ability to use existing tools in new ways, because they were designed to be used in new ways.  This is what it really means, to me, to build a working vocabulary.  Like the author mentioned in the text, every language has some set of primitives, and in order to extend that language you must define things in terms of those primitives.  As you build your working vocabulary you're naturally extending the language, and developing new programs.
+
+As a side note, even though this class focuses on language design, you can also look at this quote from the other direction. That is, when you are just doing standard coding, you are still not free to ignore the problems that language design has. Ultimately, your code will be used by other people, and will be adapted and grown into doing things it was not originally intended to do. The code you write implicitly defines a mode of thought and a way to interface human ideas with the computer. In the same way that scientists still have to learn to write well and communicate their ideas, code has to be usable and extensible. So, studying language design may well also be useful when just "coding".
 
+> If your standard is natural language, division/fraction notation isn't particularly intuitive.
+> But to someone who knows the notation, ab / b is very intuitive, and it's immediately obvious that
+> you can cancel the bs. So the goal of an intuitive programming syntax shouldn't be to emulate
+> natural language, necessarily, but to provide intuitive hooks and syntax for the types of things
+> that programmers do a lot. 
+> [Comment by "Peter" ([Source](https://www.fastcodesign.com/1665735/why-arent-computer-programming-languages-designed-better#comment-bc08f700-3c06-11e1-921b-9794fc737ae8))] [Pavlus, 2012]
+
+I feel that this comment on the Pavlus article really gets to the heart of the matter - that knowing the notation is part of any (technical) job, and once you can grok it, it becomes second nature.  So while natural language may make it easier to read and write code at first, when it comes down to it the most important thing is making sure that someone who uses it often (i.e. professionals and very passionate amateurs) can do what they need to do clearly and easily.
+
+The other point is that often, natural language is *not* actually suitable for certain purposes. Many notations are much *easier* than natural language. Usability is indeed a good aspiration, but the answer isn't so simple as "natural language". In fact, there probably isn't just one good answer. Hence why we are in a class called domain-specific languages!
```

Comment:
I totally agree with your points! What's important is that the syntax is familiar to the user--and probably also that it's not so rigid it's difficult to express oneself. Most people are familiar with their natural language, so I think it's easy to fallback on the idea that it would make great syntax for a programing language; but most people are also familiar with mathematical notation, which is for most more intuitive and simple to use when dealing with numbers.